acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/pages/PassportAPIPage.java

```diff
@@ -80,6 +82,55 @@ public void userIdentityAsJwtString(String criId, int userDataRowNumber)
         LOGGER.info("CLIENT_ID = {}", CLIENT_ID);
     }
 
+    public void getRequestToJwksEndpoint() throws IOException, InterruptedException {
+        String publicApiGatewayUrl = configurationService.getPublicAPIEndpoint();
+        LOGGER.info("getPrivateAPIEndpoint() ==> {}", publicApiGatewayUrl);
+        HttpRequest request =
+                HttpRequest.newBuilder()
+                        .uri(URI.create(publicApiGatewayUrl + "/.well-known/jwks.json"))
+                        .setHeader("Accept", "application/json")
+                        .setHeader("Content-Type", "application/json")
+                        .setHeader(
```

Comment:
Why are we passing the authorization header to the jwks endpoint?

Reply:
Looks to be inherited from a separate step will remove as you're right, authorization is not needed for the jwks endpoint